core/src/scene/sceneLoader.cpp

```diff
@@ -1545,18 +1544,28 @@ void SceneLoader::parseStyleParams(Node params, const std::shared_ptr<Scene>& sc
         } else {
             key = prop.first.Scalar();
         }
+        Node value = prop.second;
+
         if (key == "transition" || key == "text:transition") {
             parseTransition(prop.second, scene, key, out);
             continue;
         }
 
+        if (key == "texture") {
+            if (value.IsScalar()) {
+                auto strVal = value.Scalar();
+                out.push_back(StyleParam{ StyleParamKey::texture, strVal });
+            } else {
+                ;// TODO load texture
```

Comment:
Is this a documented use case, where a texture could be defined directly as a style parameter instead of in the scene and referred as a style parameter?

Reply:
Yes, I was not sure about this. Though regarding consistency we have 'Load inline material  textures' in SceneLoader.